Source/santad/EventProviders/SNTEndpointSecurityManager.mm

```diff
@@ -352,6 +354,35 @@ - (void)messageHandler:(es_message_t *)m API_AVAILABLE(macos(10.15)) {
       es_respond_auth_result(self.client, m, ES_AUTH_RESULT_ALLOW, true);
       return;
     }
+
+    case ES_EVENT_TYPE_AUTH_MOUNT: {
+      SNTConfigurator *config = [SNTConfigurator configurator];
+      if (![config blockUSBMassStorage]) {
+        es_respond_auth_result(self.client, m, ES_AUTH_RESULT_ALLOW, true);
+        return;
+      }
+
+      DASessionRef session = DASessionCreate(NULL);
```

Comment:
How about making mount logging its own event provider, and changing SNTApplication so that it can handle multiple different Event Providers? Initially, I was putting it off for this PR, but it's looking more necessary if the alternative's coupling multiple layers by passing down DASession.